nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,55 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 2.5.0
+***************
+
+:ref:`Core library <architecture>`
+==================================
+
+* The DFU result is now returned by the :c:param:`dfu_callback` callback in :c:struct:`nrf_modem_init_params` struct.
+* It is no longer necessary to call :c:func:`nrf_modem_init` twice when applying modem firmware updates.
```

Comment:
Should this figure here also be updated: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_modem/doc/delta_dfu.html#scheduling-the-firmware-update ?

Reply:
Yes, it should!